hugegraph-llm/README.md

```diff
@@ -15,69 +15,116 @@ graph systems and large language models.
 2.  Use natural language to operate graph databases (gremlin)
 3.  Knowledge graph supplements answer context (RAG)
 
+## Environment Requirements
+- python 3.8+ 
+- hugegraph 1.0.0+
+
+## Preparation
+- Start the HugeGraph database, you can do it via Docker. Refer to this [link](https://hub.docker.com/r/hugegraph/hugegraph) for guidance
```

Comment:
```suggestion
## Environment Requirements

- python 3.8+ 
- hugegraph 1.0.0+

## Preparation

- Start the HugeGraph database, you can do it via Docker. Refer to this [link](https://hub.docker.com/r/hugegraph/hugegraph) for guidance
```

Thanks for ur enhancement, we could keep the strict markdown style~

Reply:
ok